docs/releasenotes/9.4.0.rst

```diff
@@ -103,3 +103,40 @@ Added support for DDS L and LA images
 
 Support has been added to read and write L and LA DDS images in the uncompressed
 format, known as "luminance" textures.
+
+.. _restored-image-constants:
+
+Constants
+^^^^^^^^^
+
+In Pillow 9.1.0, the following constants were deprecated. Those deprecations have now
+been restored.
```

Comment:
Wonder if something like this is clearer?
```suggestion
In Pillow 9.1.0, the following constants were deprecated.
That has been reversed and these constants will now remain available.
```

Reply:
Ok, sure. I've pushed a commit changing that, and the text in deprecations.rst.